src/main/java/com/amazonaws/gurureviewercli/util/RecommendationPrinter.java

```diff
@@ -0,0 +1,82 @@
+package com.amazonaws.gurureviewercli.util;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Comparator;
+
+import lombok.NonNull;
+import lombok.val;
+import software.amazon.awssdk.services.codegurureviewer.model.RecommendationSummary;
+import software.amazon.awssdk.services.codegurureviewer.model.Severity;
+
+/**
+ * Utility class to print recommendations.
+ */
+public final class RecommendationPrinter {
+
+    private RecommendationPrinter() {
+        // do not instantiate
+    }
+
+    /**
+     * Print recommendations to command line.
+     *
+     * @param recommendations List of recommendations
+     */
+    public static void print(final Collection<RecommendationSummary> recommendations) {
+        val sortedRecommendations = new ArrayList<>(recommendations);
+        sortedRecommendations.sort(Comparator.comparing(RecommendationPrinter::severityToInt));
+        for (val recommendation : sortedRecommendations) {
+            val sb = new StringBuilder();
+            sb.append("-----\n");
+            sb.append(String.format("ID: %s, rule %s with severity %s%n",
+                                    recommendation.recommendationId(),
+                                    recommendation.ruleMetadata().ruleId(),
+                                    recommendation.severity()));
+            sb.append(String.format("In %s line %d%n", recommendation.filePath(), recommendation.startLine()));
+            sb.append(recommendation.description());
+            sb.append("\n");
+            if (severityToInt(recommendation) < 2) {
+                Log.error(sb.toString());
```

Comment:
This may look like that CLI wrapper itself hit an error. Aren't INFO messages with `Severity: FOOBAR` enough? Otherwise you're saying the same thing twice in different ways. I know this may allow loglevel-based filtering, but seems like it could also be a bit confusing.

Reply:
Yeah, you're probably right ... I'll log everything at info